support-payment-api/src/main/scala/services/CloudWatchService.scala

```diff
@@ -36,10 +37,13 @@ class CloudWatchService(cloudWatchAsyncClient: AmazonCloudWatchAsync, environmen
   def recordPaymentSuccess(paymentProvider: PaymentProvider): Unit = put("payment-success", paymentProvider)
 
   def recordFailedPayment(error: Exception, paymentProvider: PaymentProvider): Unit = {
-    if (isPaymentError(error))
+    if (isPaymentError(error)) {
+      logger.error(s"Payment error with $paymentProvider", error)
       put("payment-error", paymentProvider)
-    else
+    } else {
+      logger.info(s"Payment failure with $paymentProvider", error)
       put("failed-payment", paymentProvider)
+    }
   }
 
   def isPaymentError(error: Exception): Boolean = {
```

Comment:
Rather than matching on strings like this perhaps a better solution would be to encode `payment-error` vs `failed-payment` in the type system. It would be quite a big refactor though